### ISSUE TITLE
Lua scripts

### ISSUE THREAD

**Issue by Philippe-M** [ClickAndMortar/AdvancedCsvConnectorBundle#23]
when I want to create a lua script, I give it a name and a code (year2months), I click on "Save" but I'm stuck on this window

![image](https://user-images.githubusercontent.com/4558101/66324802-6d1a9b00-e926-11e9-9760-5d274b4aef7f.png)

In apache logs I have this error

```
[2019-10-07 17:16:05] request.INFO: Matched route "pim_customentity_rest_create". {"route":"pim_customentity_rest_create","route_parameters":{"_controller":"pim_custom_entity.controller:executeAction","actionType":"rest_create","customEntityName":"luaUpdater","_route":"pim_customentity_rest_create"},"request_uri":"http://xxxx/reference-data/rest/luaUpdater/create","method":"POST"} []
[2019-10-07 17:16:05] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ClassNotFoundException: "Attempted to load class "Lua" from the global namespace. Did you forget a "use" statement?" at /var/www/html/pim-community-standard/vendor/clickandmortar/advanced-csv-connector-bundle/Validator/LuaScriptValidator.php line 41 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ClassNotFoundException(code: 0): Attempted to load class \"Lua\" from the global namespace.\nDid you forget a \"use\" statement? at /var/www/html/pim-community-standard/vendor/clickandmortar/advanced-csv-connector-bundle/Validator/LuaScriptValidator.php:41)"} []
```

**Comment by simoncarre** [comment]
Hello @Philippe-M,

Thank you for your tests. To use LUA scripts, you need to install `php-lua` package and restart your server. I've updated the readme following your issue.

**Comment by Philippe-M** [comment]
Hello @simoncarre,

I installed php-lua, restarted apache2 and php7.2-fpm. Now I have no more errors in the logs but I'm still stuck when I want to create a script.

I restarted my browser and going through app_dev.php and I have a 404 error

![image](https://user-images.githubusercontent.com/4558101/66377150-49e8fd80-e9b1-11e9-80ec-8c0367b76c2f.png)

**Comment by Philippe-M** [comment]
Additional information. When I click on the "Save" button when creating the script. In the firefox debugger I have this error

![image](https://user-images.githubusercontent.com/4558101/66480132-3798bd80-ea9e-11e9-9c32-5699ab36c074.png)

**Comment by simoncarre** [comment]
Hello @Philippe-M,

It was a script validation error on luaUpdater creation. It's fixed.

**Comment by VikgneshvarB** [comment]
Hi @Philippe-M ,
        
This issue has been fixed in the latest patch of this bundle "1.8.19".
